nits in frames section (#517)
quicwg/multipath: draft-ietf-quic-multipath.md

## PATH_ABANDON Frame {#path-abandon-frame}

The PATH_ABANDON frame informs the peer to abandon a path.

> After the PATH_ABANDON frame is sent on a path, the path cannot be used for sending anymore.

Reviewer: Alternative

```suggestion
After the PATH_ABANDON frame is sent on a path, the path can no longer be used for sending.
```

The Retire Prior To field indicates which connection IDs
should be retired among those that share the Path ID in the Path Identifier field.
Connection IDs associated with different Path IDs are not affected.

> Note that the NEW_CONNECTION_ID frame can only be used to issue or retire
> connection IDs for the initial path with Path ID 0.
> 

Author: ```suggestion
```
Proposal in the other PR was to remove this sentence as this is already covered in the path management section.
